08-components-lab/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,69 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+/**
+ * Lab 01:
+ *
+ * Implement a time converter component.
+ * Component should show 3 input fields labeled hours, minutes and seconds.
+ * Typing a value in either field updates the other two with corresponding values.
+ */
+
+import React from 'react';
+
+export default class TimeConverter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {

```

Comment:
Do you really need 3 variables in state? after all hours, minutes and seconds have something in common. Try to use one variable in state in this case